docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -169,7 +169,7 @@ under `AppData`, not from `Program File`).
 
 ### Testing on the [dotnet/runtime](https://github.com/dotnet/runtime) repo
 
-1. make sure that you can build the `runtime` repo as baseline (run `build.cmd libs`, which should be sufficient to build all C# code, installing any prerequisites if prompted to)
+1. make sure that you can build the `runtime` repo as baseline (run `build.cmd clr+libs+libs.tests`, which should be sufficient to build all C# code, installing any prerequisites if prompted to. There's documentation on prerequisites on the runtime repo homepage. You may need to do `git clean -xdf` and `build.cmd -restore` before you can build successfully)
```

Comment:
>to. There's [](http://example.com/codeflow?start=191&length=11)

Perhaps "... to - there's ..." instead if each bullet is not a complete sentence.